src/routes/project/components/Details.js

```diff
@@ -40,8 +42,8 @@ export default class ProjectDetail extends Component {
       showMoodboard: false,
       showStoryboards: false,
       showStyleframes: false
```

Comment:
Instead of tracking all of these individually, perhaps we could just store an active section index? We could significantly reduce the complexity.